src/components/Proposals/Proposal/Proposal.jsx

```diff
@@ -7,26 +7,50 @@ import OPApprovalProposalStatus from "./OPApprovalProposalStatus";
 import ProposalTimeStatus from "./ProposalTimeStatus";
 import { cn, getProposalTypeText } from "@/lib/utils";
 import OPOptimisticProposalStatus from "./OPOptimisticProposalStatus";
+import SnapshotProposalStatus from "./SnapshotProposalStatus";
+import { ArrowTopRightOnSquareIcon } from "@heroicons/react/20/solid";
+import Tenant from "@/lib/tenant/tenant";
+import HumanAddress from "@/components/shared/HumanAddress";
 
 export default function Proposal({ proposal, votableSupply }) {
   const proposalText = getProposalTypeText(proposal.proposalType);
 
   return (
-    <Link href={`/proposals/${proposal.id}`}>
+    <Link
+      href={
+        proposal.proposalType === "SNAPSHOT"
+          ? proposal.proposalData.link
+          : `/proposals/${proposal.id}`
+      }
+      target={proposal.proposalType === "SNAPSHOT" ? "_blank" : ""}
+    >
       <HStack alignItems="center" className={styles.proposal_row}>
         <VStack className={cn(styles.cell_content, styles.cell_title)}>
-          <HStack className={styles.cell_content_title} gap={1}>
-            {/* Warning: this assumes OP FND is the only proposer. Will need to maintain an array of OP Foundation proposals eventually */}
-            <div>
-              {proposalText}{" "}
-              <span className={styles.invisible_on_mobile}>
-                by The Optimism Foundation
-              </span>
-            </div>
-            <div className={styles.mobile_status}>
-              <ProposalStatus proposal={proposal} />
-            </div>
-          </HStack>
+          {proposal.proposalType === "SNAPSHOT" ? (
+            <HStack className={styles.cell_content_title} gap={1}>
+              <p>Snapshot Proposal</p>
+              <ArrowTopRightOnSquareIcon className="w-3 h-3 mt-1" />
+            </HStack>
+          ) : (
+            <HStack className={styles.cell_content_title} gap={1}>
+              {/* Warning: this assumes OP FND is the only proposer. Will need to maintain an array of OP Foundation proposals eventually */}
+              <div>
+                {proposalText}{" "}
+                <span className={styles.invisible_on_mobile}>
+                  {Tenant.current().namespace === "optimism" ? (
```

Comment:
My preference is to start using constants for namespace flags. Something like this 
`{Tenant.current().namespace === TENANT_NAMESPACES.OPTIMISM}`